.lune/typecheck.luau

```diff
@@ -0,0 +1,16 @@
+--> Run luau-lsp analysis to check for type errors
+
+local process = require("@lune/process")
+
+local CommandBuilder = require("./lib/exec")
+
+process.exit(
+	CommandBuilder.new("~/.rokit/bin/luau-lsp")

```

Comment:
We should use pesde managed tooling for this.

Reply:
This was for testing -- I missed updating this. Fixed in https://github.com/pesde-pkg/scripts/pull/3/commits/677e49473b6cc3835a19dbe636a1217102196541.